api/versioning/versioning.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package versioning;
+
+import "google/protobuf/descriptor.proto";
+
+extend google.protobuf.MessageOptions {
```

Comment:
I believe this only exist in proto2

Reply:
I think extensions are valid in proto3 but only for annotations, see https://github.com/googleapis/googleapis/blob/master/google/api/annotations.proto for example.

Reply:
seems it is a hidden feature of proto3 (non-documented), fine either way.